### PR TITLE
Home Task 12 completed

### PR DESCRIPTION
**Pull request by Goshozavr**

### CODE REVIEW

File: lesson14/cypress-init/cypress/e2e/lesson15/dialog.spec.js [hunk 1]
```diff
@@ -19,6 +32,39 @@ describe("Dialod page", () => {
...
+          .as("Submit button check");
+
+        cy.get('nb-card[class="form-input-card"]')
+          .first()
+          .find('li[class="ng-star-inserted"]')
+          .then(($el) => {
+            const liText = "aboba";
+            cy.wrap($el)
+              .then(() => {
+                expect(liText).to.deep.equal(name);
```
Comment (sttaran) [inline lesson14/cypress-init/cypress/e2e/lesson15/dialog.spec.js:64]: `expect(liText).to.deep.equal(name);`

Comment (sttaran) [inline lesson14/cypress-init/cypress/e2e/lesson15/dialog.spec.js:64]: what the value this expect brings to your test?

File: lesson14/cypress-init/cypress/e2e/lesson15/dialog.spec.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          .click();
+
+        cy.get("@form").find('input[placeholder="Name"]').as("nameInput");
+        cy.get("@nameInput").type(name);
+        cy.get("@nameInput").invoke("val").should("be.equal", name);
+        cy.get("@form")
+          .first()
+          .find('button[status="success"]')
+          .click()
+          .as("Submit button check");
```
Comment (sttaran) [inline lesson14/cypress-init/cypress/e2e/lesson15/dialog.spec.js:55]: why? you do not use this alias

**Review by sttaran — CHANGES_REQUESTED** [review]

File: lesson14/cypress-init/cypress/e2e/lesson15/dialog.spec.js (hunk 2 above)
Comment (Goshozavr) [reply lesson14/cypress-init/cypress/e2e/lesson15/dialog.spec.js:55]: It's just to highlight the button check)

File: lesson14/cypress-init/cypress/e2e/lesson15/dialog.spec.js (hunk 1 above)
Comment (Goshozavr) [reply lesson14/cypress-init/cypress/e2e/lesson15/dialog.spec.js:64]: It's an assertion to check if the final text in the list matches provided one (name) 
actually - this is an expected result for this e2e test, isn't it?

Comment (Goshozavr) [reply lesson14/cypress-init/cypress/e2e/lesson15/dialog.spec.js:64]: I didn't get what's wrong here

File: lesson14/cypress-init/cypress/e2e/lesson15/dialog.spec.js (hunk 2 above)
Comment (Goshozavr) [reply lesson14/cypress-init/cypress/e2e/lesson15/dialog.spec.js:55]: I saw that the aliases are highlights during test execution, so I decided to do it)

**Pull request merged**